include/api.php

```diff
@@ -3664,6 +3667,336 @@ function api_friendica_notification_seen($type){
 	api_register_func('api/friendica/notification', 'api_friendica_notification', true, API_METHOD_GET);
 
 
+	/**
+	 * @brief return direct_messages for Windows 10 App (similar to direct_messages/all, but seen 
+	 * and parent-uri added to output
+	 *
+	 * @param App $a
+	 * @param string $type Known types are 'atom', 'rss', 'xml' and 'json'
+	 * @return string
+	 */
+	function api_friendica_direct_messages_all($type){
```

Comment:
Couldn't this call be replaced by the existing API call with maybe an additional parameter?